package.json

```diff
@@ -67,7 +67,7 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "^1.1.0",
-    "@edx/frontend-build": "^12.0.0",
+    "@edx/frontend-build": "12.0.3",
```

Comment:
Curious to know the decision to pin this version versus the caret range?
We don't have auto merges for dependencies upgrades in this repo with Renovate.

Reply:
I was doing frontend-build upgrade for all the mfes, pinning frontend-build version was based on feedback I received on some other PRs. For this repo I don't see an issue using caret range.